tests/test_bot.py

```diff
@@ -1496,6 +1443,66 @@ def localize(dt, tzinfo):
             # fail if this branch is ever reached.
             sys.exit(1)
 
+    test_flag = None
```

Comment:
please move up directly beneath the docstring. having this below a method is confusing …

Reply:
was moved to a local variable (inside the test method) instead.